service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/Descriptor.java

```diff
@@ -390,14 +390,14 @@ public String toString() {
     /**
      * Describes a topic call.
      */
-    public static final class TopicCall<Message> {
+    public static final class TopicCall<MESSAGE> {
```

Comment:
If you guys are still against this change I can revert it. Another option would be to move closer to the standard by making it shorter like `REQ` and `RSP` or something. I agree that just `R` would be too confusing in this case.